Update w/ new mail guidelines (#23)
hackclub/toolbox: content/stickers.mdx

## How to request sticker shipments

1. Sign into the Hack Club Slack
2. Fill out this form: <Code>recowYSV2wClqHBIo</Code> (if you are in India, DM [@Harsh Bajpai](https://hackclub.slack.com/team/U010XUNLX40))
3. Make sure to include your address (if you're sending stickers to someone else, include their full address)
4. Let us know if the stickers are just for 1 recipient, or for a whole club. If it's for a club or hackathon, include your attendance count.

<h3 id="responsibility">With great power comes great responsibility</h3>

> This is a system built on trust. Every shipment request is manually sent by someone from Mail Team, and every sticker shipment is personally prepared, packed, and mailed.

Reviewer: ```suggestion
This is a system built on trust. Every shipment request is manually sent by someone at HQ, and every sticker shipment is personally prepared, packed, and mailed.
```

I consider Mail Team apart of HQ
